agent/src/main/java/com/microsoft/hydralab/agent/runner/ActionExecutor.java

```diff
@@ -28,7 +28,8 @@ public class ActionExecutor {
     /**
      * the implementation of supported actions should not be overload
      */
-    private Set<String> actionTypes = Set.of("setProperty", "setDefaultLauncher", "backToHome", "changeGlobalSetting", "changeSystemSetting");
+    private Set<String> actionTypes = Set.of("setProperty", "setDefaultLauncher", "backToHome", "changeGlobalSetting",
```

Comment:
Suggest we migrate this to application.yml and make it configurable at the agent level, and the agent admin can specify this themselves.

We may do this in another PR.

Reply:
Will do this work in another PR. If use application.yml to config, we need to add ActionExecutor to bean factory.